packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -66,6 +66,18 @@ export interface TableProps {
     writeAutoScaling?: AutoScalingProps;
 }
 
+export interface AttributeProps {
```

Comment:
I think this might better be called `Attribute`. `Props` is usually reserved for the set of parameters we use to instantiate a new class, which this is not.

Reply:
I will rename it in the next revision.